### PR TITLE
Implemented NthPrime (`nth-prime`) exercise for objective-c

### PR DESCRIPTION
**Pull request by mindejulian**
Implemented NthPrime (`nth-prime`) exercise for objective-c.

### CODE REVIEW

File: exercises/nth-prime/NthPrimeTest.m [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+NS_ASSUME_NONNULL_BEGIN
+
+@interface NthPrimeTest : XCTestCase
+
+@end
+
+@implementation NthPrimeTest
+
+- (void)testFirstPrime {
+	int primeNo = 1;
```
Comment (masters3d) [inline exercises/nth-prime/NthPrimeTest.m]: What do you think about changing all of these to `primeNum`?

File: exercises/nth-prime/NthPrimeExample.h [hunk 2]
```diff
@@ -0,0 +1,6 @@
+@interface NthPrime : NSObject {
+
+}
++(int)primeNo:(int)primeNo;
```
Comment (masters3d) [inline exercises/nth-prime/NthPrimeExample.h]: This is cool but at a glance it looks like prime no. Do you mind changing?

File: exercises/nth-prime/NthPrimeTest.m (hunk 1 above)
Comment (mindejulian) [reply exercises/nth-prime/NthPrimeTest.m]: I meant it as short for prime number all along, but I do agree that `primeNum` is better.

**Comment by masters3d** [comment]
Thanks for this, much appreciated! Our Objc track doesn't get as much love as our Swift one :)

**Pull request merged**